dashboard/test/controllers/congrats_controller_test.rb

```diff
@@ -28,7 +28,10 @@ class CongratsControllerTest < ActionController::TestCase
   test "cached query test for hoc course" do
     hoc_course = create(:hoc_course)
 
-    Unit.stubs(:should_cache?).returns(true)
+    setup_script_cache
+    UnitGroup.stubs(:should_cache?).returns true
+    @@course_cache ||= UnitGroup.course_cache_to_cache
+    UnitGroup.course_cache
```

Comment:
Following the example from setup_script_cache, I think this can just be:
```suggestion
    UnitGroup.clear_cache
```
the stub is unnecessary because the stubbed method refers to Unit.should_cache? which has already been stubbed. the other two lines are effective because they get rid of the old contents of the cache, but there's no reason to load the new contents because assert_cache_queries takes care of making sure the cache is populated for the records we care about by running the block twice.